content/remote-development/getting-started.md

```diff
@@ -28,6 +28,13 @@ If the GitHub repository is not the origin of your checked out project then you
 
 ### OSX (64-bit):
 
+If you use [Homebrew](https://brew.sh/), you can install cp-remote via:
+
+```
+brew install inviqa/tools/cp-remote
```

Comment:
can do `continuouspipe/tools/cp-remote, but you'd need to create a `continuouspipe/homebrew-tools` repo